packages/web-frontend/src/containers/MapOverlayBar/TitleAndDatePicker.js

```diff
@@ -0,0 +1,157 @@
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import moment from 'moment';
+import { withStyles } from '@material-ui/core/styles';
+import DownArrow from '@material-ui/icons/ArrowDropDown';
+import MuiSwitch from '@material-ui/core/Switch';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { DateRangePicker } from '../../components/DateRangePicker';
+import { getDefaultDates, getLimits, GRANULARITY_CONFIG } from '../../utils/periodGranularities';
+import { Content, ContentText } from './Styles';
+import { TUPAIA_ORANGE } from '../../styles';
+import { setDisplayedMapOverlay } from '../../actions';
+
+const Switch = withStyles({
```

Comment:
Not sure if we need `withStyles` here -perhaps we can use `styled` instead?

Reply:
I was thinking to use `styled` but that's how material ui want us to do for switch's custom colour: https://v4.mui.com/components/switches/#CustomizedSwitches.js 

I haven't tried very hard on use `styled`, but I think that should be fine to use `withStyles`?

Reply:
I believe that material-ui exposes its own custom styling helper (`withStyles`), but we generally prefer `styled` over it.

Please give `styled` a go, if it's too tricky then let's stick with `withStyles`

Reply:
Switch use it's only logic to render check and uncheck colour, seems like styled can't easily override it's colour.. Or perhaps I make something wrong on css. Would you like to pair on it?

Reply:
Yeap, happy to have a a look!